File: Group 57/level 32/Homework/hw2.py
# 3) გააკეთეთ Filter Even Numbers. მიზანი: გაფილტრეთ ყველრა ლუწი რიცხვი და ჩაამატეთ ახალ სიაში და შემდეგ ეგ სია დაპრინტეთ 

numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
even_numbers = []

i = 0

while i < len(numbers):
    if numbers[i] % 2 == 0:
        even_numbers.append(numbers[i])
    i += 1

print(even_numbers)

File: Group 57/level 101/Homework/hw2.js
/* 2)მომხმარებელს შემოატანინე ორი რიცხვი,მომხმარებლის მიერ შემოტანილი 
პირველი რიცხვიდან მეორე რიცხვამდე იპოვეთ ყველა ლუწი რიცხვის ჯამი */

const start = Number(prompt("Enter first number:"))
const end = Number(prompt("Enter second number:"))

let sum = 0

for (let i = start; i <= end; i++) {
    if (i % 2 === 0) {
        sum += i
    }
}

console.log("even numbers sum:", sum)